packages/kbn-optimizer/limits.yml

```diff
@@ -1,5 +1,6 @@
 pageLoadAssetSize:
   advancedSettings: 27596
+  actions: 20000
```

Comment:
The actual value was ~16K, figured I'd add some slop, but not quite sure if that's ok